mmv1/products/iam3/PrincipalAccessBoundaryPolicy.yaml

```diff
@@ -41,12 +46,20 @@ async:
   include_project: true
 examples:
   - name: 'iam_principal_access_boundary_policy'
+    primary_resource_id: 'pab-policy-for-org'
+    test_env_vars:
+      org_id: 'ORG_ID'
+    vars:
+      display_name: 'PAB policy for Organization'
+      pab_id: 'pab-policy-for-org'
+  - name: 'iam_organizations_policy_binding'
```

Comment:
The test is failing because this block also needs a `external_providers: ["time"]` but really you can just delete the test altogether